### PR TITLE
better readability of logs

### PR DESCRIPTION
**Pull request by NoSkillGuy**
when running zboxcli, logs look like this 
```
0chain-core-sdk [INFO]   2021/06/21 02:43:31.089961 wallet.go:276: ******* Wallet SDK Version:v1.2.6 *******
0box-sdk [INFO]   2021/06/21 02:43:31.090784 sdk.go:69: ******* Storage SDK Version: v1.2.6 *******
0chain-core-sdk [INFO]   2021/06/21 02:43:31.161371 wallet.go:359: *******  Wallet SDK Version:v1.2.6 *******
0box-sdk [DEBUG]  2021/06/21 02:43:31.175406 networkworker.go:98: Get network result:{"miners":["http://198.18.0.71/","http://198.18.0.72/","http://198.18.0.74/","http://198.18.0.73/"],"sharders":["http://198.18.0.81/"]}
```

with this small change, logs would look like this. 
```
0chain-core-sdk [INFO]   2021/06/21 03:09:32.861001 wallet.go:276: ******* Wallet SDK Version:v1.2.6 *******
0box-sdk        [INFO]   2021/06/21 03:09:32.861517 sdk.go:70: ******* Storage SDK Version: v1.2.6 *******
0chain-core-sdk [INFO]   2021/06/21 03:09:32.880253 wallet.go:359: *******  Wallet SDK Version:v1.2.6 *******
0box-sdk        [DEBUG]  2021/06/21 03:09:32.889983 networkworker.go:98: Get network result:{"miners":["http://198.18.0.71/","http://198.18.0.72/","http://198.18.0.74/","http://198.18.0.73/"],"sharders":["http://198.18.0.81/"]}

```

### CODE REVIEW

**Review by moldis — CHANGES_REQUESTED** [review]
It's good idea, but I don't think adding extra spaces in Init is a flexible solution.

**Comment by NoSkillGuy** [comment]
@moldis completely agreed on this. This change is not far-sighted. 
I have searched all the projects in 0chain, for `Logger.Init` , it's used in only two places. so this is a quick hack! 

I thought of a better approach and created a sync functionality in logger package.

File: core/logger/logger.go [hunk 1]
```diff
@@ -65,6 +65,26 @@ func (l *Logger) SetLevel(lvl int) {
 	l.lvl = lvl
 }
 
+// syncPrefixes - syncs the logger prefixes
```
Comment (moldis) [inline core/logger/logger.go:68]: It's private function, comment not needed

Comment (NoSkillGuy) [reply core/logger/logger.go:68]: Got it, but having comments for every function, helps in the readability of code right!

**Review by moldis — APPROVED** [review]
Looks good

**Pull request merged**